### PR TITLE
Recs Sync Validation

### LINKED ISSUE #70: Recs `sync` Validation
When running the `recs sync` command, the branch name can be provided as an option. It is possible however to provide this incorrectly which will create a template with the wrong configuration.

Example of incorrect command input:
```
snapfu recs sync standard-template branch=production
```

^ this assumes that the `branch` is `branch=production` - we should add some input validation logic for this.

### PR DESCRIPTION
**Pull request by chrisFrazier77**

### CODE REVIEW

**Review by korgon — COMMENTED** [review]
Nice. Can we also add the same check for the `recs archive` command?

File: src/recs.js [hunk 1]
```diff
@@ -215,6 +215,10 @@ export async function removeTemplate(options) {
 	}
 
 	const branchName = branch || repository.branch || DEFAULT_BRANCH;
+	if (!repository.branchList.includes(branchName)) {
```
Comment (korgon) [inline src/recs.js]: Might want some optional chaining here incase branchList is undefined (where it is defined is within a try/catch that does nothing).

**Review by korgon — CHANGES_REQUESTED** [review]
Small change request that ought to make code less likely to error out.

File: src/recs.js (hunk 1 above)
Comment (chrisFrazier77) [reply src/recs.js]: Not sure why but the optional chaining was not working, it was throwing errors at the ? characters. So I just did it the old fashion way. 
B9727486-4E41-499A-9EA4-5D9890A38A9E_4_5005_c

Comment (korgon) [reply src/recs.js]: Yea, I think we allowed for this in the `patch` branch (merged into Develop). Old fashioned is good for now :)

**Review by korgon — APPROVED** [review]

**Pull request merged**